plugins/obs-nvenc/data/locale/en-US.ini

```diff
@@ -53,6 +53,7 @@ SplitEncode.ThreeWay="Three-way split"
 
 Opts="Custom Encoder Options"
 Opts.TT="Space-separated list of options to apply to the rate control and codec settings,\nbased their names in the nvEncodeAPI header.\ne.g. \"lookaheadDepth=16 aqStrength=4\""
+Opts.Invalid="Invalid Custom Encoder options, check the log for details.\n\nSee our <a href=\"https://obsproject.com/kb/advanced-nvenc-options\">Knowledge Base Article</a> for information on available options."
```

Comment:
For me, the `<a>` tag is not rendering as a hyperlink, it just renders as plain text: `<a href=\"https://obsproject.com/kb/advanced-nvenc-options\">Knowledge Base Article</a>`

Reply:
Yeah I forgot to push the translation string fixes. Should be fine now.